Source/Core/Core/IOS/ES/ES.cpp

```diff
@@ -241,7 +242,24 @@ bool ES::LaunchTitle(u64 title_id, bool skip_reload)
   // (supposedly when trying to re-open those files).
   DiscIO::NANDContentManager::Access().ClearCache();
 
-  if (IsTitleType(title_id, IOS::ES::TitleType::System) && title_id != TITLEID_SYSMENU)
+  u32 device_id;
+  constexpr u32 DEFAULT_DEVICE_ID = 0x0403ac68;
```

Comment:
I dislike duplicating magic constants like this, especially when theres no record of where else it might be used. I wasn't able to find the other instance (by looking, not by searching) so theres that.

Maybe at least some comments (in both places) that reminds other people about the other location if you don't want it in a header?